### DISCUSSION TITLE
Without Chinese?

### DISCUSSION THREAD

**Discussion by tidexe** [tobychui/arozos#64] [Q&A]
How should I set up Chinese? Or no Chinese?

**Comment by tobychui** [comment]
Hi, there is no Chinese support for now due to lack of man power developing this system.
If you wanna help translate the system, feel free to contact me or create a PR for the translation files. Thanks!

**Reply by super-soft** [reply]
Well，I really like this web-desktop system,  and I think it is the best cloud system from linux.but please create Chinese language thanks a lot~~

**Reply by super-soft** [reply]
It is wonderful , but only don`t support Chinese. ArozOS is very important for me , and many Chinese , so I hope that ArozOS will support Chinese one day.

**Reply by tobychui** [reply]
@super-soft This discussion is a bit old. We did have partial Chinese support on some Apps now (mostly File Manager and its related functions, see the screenshot below). 

![圖片](https://user-images.githubusercontent.com/24617523/178718212-8a605fdc-0ced-46fb-9cd3-0c135fc74a46.png)

If your are interested to enable (partial) localization on your system, you can change your system language setting by going to System Setting > Device > Language
![圖片](https://user-images.githubusercontent.com/24617523/178718544-baaa80ef-3906-45a8-b81d-3c337dd4f911.png)

And pick your preferred language. If no prefer is set, it will use your browser default instead. 
![圖片](https://user-images.githubusercontent.com/24617523/178718571-5263527d-9156-4c5c-a4e4-891e5852b095.png)

However, as stated above, we are still lacking man power to help localize this system. If you know a bit HTML and JavaScript, feel free to contact me via Telegram or Email and I can guide you through how to help with the location of this system :)
